### ISSUE TITLE
TIme to Act

### ISSUE THREAD

**Issue by pramitchoudhary** [Mondego/TubeKit#2]
Need to set new milestones. No action for quite sometime. Time to get this project in full swing.

**Comment by shahc** [comment]
Agree. We should first check if all the components are working fine. These include YouTube APIs compatibility as well as Firefox toolbar. If it's possible, the system needs to be tested on Windows 7,8 systems and documentation needs to be updated.

**Comment by pramitchoudhary** [comment]
Yup, let me first make sure all the existing components are working as expected. Proper documentation on the known issues.

**Comment by JorgenPhi** [comment]
Any activity (or interest) on this project? Would love to have something like this modernized.

**Comment by shahc** [comment]
There is interest (I hear from people time to time), but no activities recently. Perhaps Pramit can chime in.

**Comment by pramitchoudhary** [comment]
Hi All,
     Most certainly, the project has a lot of scope and some of the initial
work done by Prof Shah is incredible. This last couple 8-9 months has been
very hectic for me wrt job and life in general. I have been wanting to get
back to the project so many times.
Let me try one more time this weekend to see if I can at-least get the
existing parts working. Then I can write a proposal for changes and Prof.
Shah and others can help in reviewing the same. Does that sound good ?

Regards,
Pramit

On Mon, Dec 1, 2014 at 12:30 PM, Chirag Shah notifications@github.com
wrote:

> There is interest (I hear from people time to time), but no activities
> recently. Perhaps Pramit can chime in.
> 
> > On Dec 1, 2014, at 3:16 PM, Jorgen Phillips notifications@github.com
> > wrote:
> > 
> > Any activity (or interest) on this project? Would love to have something
> > like this modernized.
> > 
> > —
> > Reply to this email directly or view it on GitHub <
> > https://github.com/Mondego/TubeKit/issues/2#issuecomment-65127450>.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/Mondego/TubeKit/issues/2#issuecomment-65129381.

## 

Pramit Choudhary
949.864.9717
www.linkedin.com/in/pramitc/

**Comment by shahc** [comment]
Sounds good. Keep me updated!
